osu.Framework/Configuration/ConfigManager.cs

```
@@ -122,19 +122,20 @@ public U Get<U>(T lookup)
             return GetBindable<U>(lookup).Value;
         }
 
-        public Bindable<U> GetBindable<U>(T lookup)
+        protected Bindable<U> GetBindable<U>(T lookup)

```

Unsure about this method returning null if U does not match the type of the bindable that lookup refers to. Maybe instead throw an exception informing about the type mismatch (and add a separate method to check for this condition, if we actually want to meaningfully use the functionality of detecting whether those types match, although imho this seems obscure).